### DISCUSSION TITLE
Too long status bar in nw mode

### DISCUSSION THREAD

**Discussion by signifex** [syl20bnr/spacemacs#15723] [Q&A]
Im using spacemacs in nw mode, and status bar is always 2-5 characters longer, than terminal. In tmux pane this line goes to pane that right from spacemacs. But the resolution of the rest of the window is normal.

**Comment by lebensterben** [comment]
Screenshot?

**Reply by signifex** [reply]
already solved: I changed ...line-unicode... to nil

![image](https://user-images.githubusercontent.com/97762325/188519448-2a42950d-cc68-4db5-b655-31ac82ae62e0.png)
![image](https://user-images.githubusercontent.com/97762325/188519479-63fa54ec-b5a5-426f-81e4-16f66e675fbd.png)
